src/YQTimezoneSelector.cc

```diff
@@ -243,7 +242,9 @@ void YQTimezoneSelector::paintEvent( QPaintEvent *event )
         }
         p.drawPixmap( ( width() - d->cachePix.width() ) / 2, ( height() - d->cachePix.height() ) / 2, d->cachePix );
 
-        setCursor( QCursor( QPixmap( zoom_in ) ) );
+        Q_INIT_RESOURCE(qt_icons);
+        QIcon icon = QIcon::fromTheme( "zoom-in", QIcon( ":/zoom-in" ) );
+        setCursor( QCursor( icon.pixmap( QSize( 16, 16 ) ) ) );
```

Comment:
If loading that pixmap failed, we should rather not set a cursor at all and keep the default cursor. I fear like this it will simply use an empty pixmap, so the cursor will not be visible.